etc/openwrt/openthread-br/README.md

```diff
@@ -0,0 +1,44 @@
+OpenThread Border Router on OpenWRT
+===================================
+
+## Get Started
+
+This is for local development.
+
+1. Add OpenThread feed.
```

Comment:
I would go ahead and make each major step level 3 headers so they stand out more

### 1. Add the OpenThread feed
### 2. Enable OpenThread
etc